packages/tinacms/src/tina-cms.tsx

```diff
@@ -212,48 +180,63 @@ const parseURL = (url: string): { branch; isLocalClient; clientId } => {
   }
 }
 
-export const TinaCMSProvider2 = ({
-  children,
-  cmsCallback,
-  mediaStore,
-  tinaioConfig,
-  ...props
-}: {
-  /** The query from getStaticProps */
-  query?: string
-  /** Any variables from getStaticProps */
-  variables?: object
-  /** The `data` from getStaticProps */
-  data: object
-  /** Your React page component */
-  children: () => React.ReactNode
-  /**
-   * The URL for the GraphQL API.
-   *
-   * When working locally, this should be http://localhost:4001/graphql.
-   *
-   * For Tina Cloud, use https://content.tinajs.io/content/my-client-id/github/my-branch
-   */
-  apiURL: string
-  /**
-   * Point to the local version of GraphQL instead of tina.io
-   * https://tina.io/docs/tinacms-context/#adding-tina-to-the-sites-frontend
-   *
-   * @deprecated use apiURL instead
-   */
-  isLocalClient?: boolean
-  /**
-   * The base branch to pull content from. Note that this is ignored for local development
-   *
-   * @deprecated use apiURL instead
-   */
-  branch?: string
-  /**
-   * Your clientID from tina.aio
-   *
-   * @deprecated use apiURL instead
-   */
-  clientId?: string
+type APIProviderProps =
+  | {
+      /**
+       * Content API URL
+       *
+       * @deprecated use apiURL instead
+       */
```

Comment:
This is a bit confusing, I'm suspecting it was just a bit of overzealous copy-paste?

Reply:
whoops! yup just removed the deprecated tag here